docs/auth0_apis_list.md

```diff
@@ -24,7 +24,7 @@ auth0 apis list [flags]
 
 ```
       --json         Output in json format.
-  -n, --number int   Number of apps to retrieve (default 50)
+  -n, --number int   Number of APIs, that match the search criteria, to retrieve. Maximum result number is 1000. (default 50)
```

Comment:
I'm seeing that _all_ your descriptions follow the same pattern. I don't necessarily agree with the phrasing of "that match the search criteria" because it suggests the ability to query based on some properties. That is certainly true for a limited number of resources but not for all. Let's keep the phrasing simple.

Reply:
Changed help text in d8c07f88597419f779b354ed158439265d41613a